packages/m2m-gateway/test/api/tenants/getTenants.test.ts

```diff
@@ -0,0 +1,110 @@
+import { describe, it, expect, vi } from "vitest";
+import { generateToken } from "pagopa-interop-commons-test";
+import { AuthRole, authRole } from "pagopa-interop-commons";
+import request from "supertest";
+import { m2mGatewayApi } from "pagopa-interop-api-clients";
+import { generateId } from "pagopa-interop-models";
+import { api, mockTenantService } from "../../vitest.api.setup.js";
+import { appBasePath } from "../../../src/config/appBasePath.js";
+
+describe("GET /tenants route test", () => {
```

Comment:
Missing integration tests! I would still add them even if you don't throw errors or perform any polling in this call, to test that the clients are called with the right params - see for example https://github.com/pagopa/interop-be-monorepo/pull/1831/files#diff-feba3f0d54345bc30bbb9bb82be3a3b325758e6c67574802e6d5665e29a2daecR62